.github/workflows/main.yml

```diff
@@ -0,0 +1,13 @@
+name: CI
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+jobs:
+  build:
+    name: Unit tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ./tests/CI/run-unit-test
```

Comment:
That would be great to add a name on each of these steps. 
Here's an example: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idsteps .

Reply:
I added names